File: snaker/app/domain/message/output.py
from pydantic import BaseModel

from snaker.app.domain.message.audio import ObjectPdn


class OutputMessage(BaseModel):
    request_id: int
    old_file_path: str
    new_file_path: str
    original_text: str
    anon_text: str
    objects_pdns: list[ObjectPdn]

File: gopher/internal/infrastructure/kafka/message/output.go
package message

type ObjectPdn struct {
	Text      string  `json:"text"`
	Type      string  `json:"type"`
	StartTime float64 `json:"start_time"`
	EndTime   float64 `json:"end_time"`
}

type OutputMessage struct {
	RequestID    int         `json:"request_id"`
	OldFilePath  string      `json:"old_file_path"`
	NewFilePath  string      `json:"new_file_path"`
	OriginalText string      `json:"original_text"`
	AnonText     string      `json:"anon_text"`
	ObjectsPdns  []ObjectPdn `json:"objects_pdns"`
}
